kvbc/include/v4blockchain/v4_blockchain.h

```diff
@@ -86,6 +87,9 @@ class KeyValueBlockchain {
   const v4blockchain::detail::Blockchain &getBlockchain() const { return block_chain_; };
   const v4blockchain::detail::StChain &getStChain() const { return state_transfer_chain_; };
   const v4blockchain::detail::LatestKeys &getLatestKeys() const { return latest_keys_; };
+  v4blockchain::detail::VerifiabilityManagementLayer &getVerifiabilityManagementLayer() {
```

Comment:
Shouldn't this be optional? (Ethereum only?)

Reply:
The actual processor of the state will be instantiated based on configuration. See SyncMerkleProcessor.h and EmptyProofProcessor.h